addons/index.md

```diff
@@ -8,15 +8,67 @@ layout: documentation
-| [3rd Party System Integration](io.html) | Expose openHAB to external systems |
+There are several ways you can install an add-on. 
+These are described under *Installation of Add-ons* below
+
+| Add-on Type                             | Description                                                                                                               |
+|-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------|
+| [Bindings](bindings.html)               | Bindings integrate physical hardware, external systems and web services in openHAB                                        |
+| [User Interfaces](uis.html)             | User interfaces are either native smartphone apps or web applications that access the openHAB server through the REST API |
+| [Persistence](persistence.html)         | Persistence services allow openHAB to store time series data for history-based actions or statistics                      |
+| [Actions](actions.html)                 | Actions are predefined methods for openHAB rules and scripts                                                              |
+| [Transformations](transformations.html) | Transformations are used to translate between technical and human-readable values for items                               |
+| [Voice Services](voices.html)           | Services that provide voice enabling features, such as text-to-speech, speech-to-text etc.                                |
+| [3rd Party System Integration](io.html) | Expose openHAB to external systems                                                                                        |
+
+## Installation of Add-ons
+
+Depending on the [package]({{base}}/configuration/packages.html) you have choosen during your first time setup, there are already some pre-installed add-ons.
+Additional add-ons can be installed in the different ways, described below.
+
+### Through Paper UI
+
+Navigate to the add-ons section.
+Search for the desired add-on in the categories and press install.
+
+![installing Add-ons through Paper UI](images/index_installation_paperui.gif)
+
+### From the openHAB Console
+
+Connect to the [openHAB console]({{base}}/administration/console.html) in your prefered [way]({{base}}/administration/console.html#accessing-the-console).
+Use either the [bundle]({{base}}/administration/bundles.html) command family to install a specific add-on:
+
+```sh
+bundle:install <URL-TO-ADD-ON-JAR-FILE>
+```
+
+You can find valid bundle `.jar` files on [cloudbees](https://openhab.ci.cloudbees.com) for example.
+Just navigate to your desired add-on through the cloudbees page and copy the
+url of the corresponding `.jar` file.
+
+![installing Add-ons from Karaf console](images/index_installation_karaf.gif)
+
+Or you could also use the feature command family to install a specific add-on:
+
+```sh
+feature:install <feature>[/version]
```

Comment:
After thinking about it, i would vote against mentioning it then.

This should be a help and a would want to prevent especially basic users with several options to choose and in detail tell them "with this option you have to pay attention to a bug or something similar".

If one reads how to install an add-on and chooses an option, this option should work properly then.

Reply:
Agree, so let's first have the issue fixed and then add this option to the docs.

Reply:
Agreed